test/simple_test/simple_test.c

```diff
@@ -0,0 +1,320 @@
+/** \file
+ * \brief Example code for Simple Open EtherCAT master
+ *
+ * Usage: simple_test IFNAME1
+ * IFNAME1 is the NIC interface name, e.g. 'eth0'
+ *
+ * This is a minimal test.
+ */
+
+#include "ethercat.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+
+typedef struct {
+    ecx_contextt    context;
+    char *          iface;
+    uint8           group;
+    int             roundtrip_time;
+
+    /* Used by the context */
+    uint8           map[4096];
+    ecx_portt       port;
+    ec_slavet       slavelist[EC_MAXSLAVE];
+    int             slavecount;
+    ec_groupt       grouplist[EC_MAXGROUP];
+    uint8           esibuf[EC_MAXEEPBUF];
+    uint32          esimap[EC_MAXEEPBITMAP];
+    /* I would say this needs to be elist[EC_MAXELIST],
```

Comment:
Why, a single ring of EC_MAXELIST error entries

Reply:
I don't know what I'm talking about, but that the field name and the existence of `EC_MAXELIST` strongly suggested that `elist` should be an array (similar to e.g. `grouplist`) although it was not used in that way by the sample code.
  
A recent look at [`ec_eringt`](https://github.com/OpenEtherCATsociety/SOEM/blob/d548e9907db8ba2e4c0967f4769734255f97f371/soem/ethercatmain.h#L346-L352) shown me how this is really implemented, so this is just a case of unfortunate naming.

I will remove the comment.